.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - "3.6"
+  - "3.7"
+# command to install dependencies
+env:
+  matrix:
+  - PYMATGEN_VERSION=2019.1.13
```

Comment:
we do not need this environment variable

Reply:
fixed in pr #4